### PR TITLE
Provide commands to get AEM status information (SKYOPS-36712/SKYOPS-54405)

### PR DESCRIPTION
**Pull request by nonanalou**
## Description
Provide commands to get AEM status information.

**USAGE**
```
$ aio aem rde inspect COMMAND
```
**TOPICS**
 ` aem rde inspect request-logs ` Get the list of request-logs for the target of a rapid development environment.

**COMMANDS**
` aem rde inspect setup `             Set the authorization token for all the inpect comands.
` aem rde inspect inventory `             Get the list of inventories for the target of a rapid development environment.
` aem rde inspect logs`                        Get the list of logs for the target of a rapid development environment.
` aem rde inspect osgi-bundles`         Get the list of osgi-bundles for the target of a rapid development environment.
` aem rde inspect osgi-components`  Get the list of osgi-components for the target of a rapid development environment.
`aem rde inspect osgi-configurations`  Get the list of osgi-configurations for the target of a rapid development environment.
`aem rde inspect osgi-services`        Get the list of osgi-services for the target of a rapid development environment.
`aem rde inspect request-logs`         Get the list of request-logs for the target of a rapid development environment.
`aem rde inspect sling-requests`       Get the list of sling-requests for the target of a rapid development environment.

## Related Issue
[SKYOPS-54405](https://jira.corp.adobe.com/browse/SKYOPS-54405) Enable aio authentication for AEM requests
[SKYOPS-36712](https://jira.corp.adobe.com/browse/SKYOPS-36712) Provide API for AEM status information

## How Has This Been Tested?

`npm test`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,19 +38,52 @@ The plugin needs to be configured to point to an existing RDE environment as fol
 > **Note**:
 > If you are planning to work with multiple environments, we highly recommend you to use the flag `-l` or `--local` together with the `config:set` so that you will store the configuration in the local directory (i.e. the config is only effective in the current directory). For details on different config locations refer to [aio-lib-core-config's README](https://github.com/adobe/aio-lib-core-config#persistent-file-locations).
 
+## Configuration for `aio aem rde inspect` commands
+
+### Enable `aio aem rde inspect` commands
```
Comment (jsedding) [inline README.md]: Either remove this section or add a fat warning that experimental features may not work, may not (yet) be available and may be removed without notice.

File: package.json [hunk 2]
```diff
@@ -57,9 +58,9 @@
   "oclif": {
     "commands": "./src/commands",
     "bin": "aio",
```
Comment (jsedding) [inline package.json:60]: You need to mark "aem:rde:inspect" as an experimental feature.

```suggestion
    "bin": "aio",
    "experimental-features": ["aem:rde:inspect"], 
```

File: package.json [hunk 3]
```diff
@@ -57,9 +58,9 @@
...
-    "devPlugins": [
-      "@oclif/plugin-help"
-    ],
+    "hooks": {
+      "init": "./src/lib/hooks/experimental-features-init-hook"
+    },
     "topics": {
       "aem:rde": {
```
Comment (jsedding) [inline package.json:66]: Please add a description for the "aem:rde:inspect" topic. Currently, I get the description of the "aem:rde:inspect:inventory" command when running `aio aem rde` (default must be the description of the first command?).

**Review by jsedding — CHANGES_REQUESTED** [review]
@nonanalou The PR looks good! I left a few comments, could you please check them? Thanks!

File: README.md (hunk 1 above)
Comment (nonanalou) [reply README.md]: Reloved: [692fc0b](https://github.com/adobe/aio-cli-plugin-aem-rde/pull/46/commits/692fc0b8dc6146f9de114c9e9b4d697ca059ed17)

File: package.json (hunk 3 above)
Comment (nonanalou) [reply package.json:66]: Resolved: [a71ffc5](https://github.com/adobe/aio-cli-plugin-aem-rde/pull/46/commits/a71ffc521adb776ea74e8665845622cfb6cd97a5)

File: package.json (hunk 2 above)
Comment (nonanalou) [reply package.json:60]: @jsedding should i already enable it? because like that it is not enabled for me.

File: package.json (hunk 3 above)
Comment (jsedding) [reply package.json:66]: Thank you!

File: package.json (hunk 2 above)
Comment (jsedding) [reply package.json:60]: No, it should not yet be enabled by default.

**Review by jsedding — APPROVED** [review]
Thanks for addressing my comments! It looks good now!

**Pull request merged**